model/vm_host.rb

```diff
@@ -251,14 +251,24 @@ def init_health_monitor_session
 
   def check_pulse(session:, previous_pulse:)
     reading = begin
```

Comment:
re, commit message:

> 2. `smartctl` binary not found:
>    - This could be due to disk detachment. We're currently ignoring this
>      scenario.
>    - Reasoning: The smartctl binary's page should be in memory since we
>      run it every 30 seconds. However, we can't make assumptions about
> client usage.
>    - Potential solution: Pin the page to RAM using `vmtouch`.

Is that actually a failure mode that exists? Is a file system really going to detach references to files because of a storage device gone bad, rather than yield an EIO errno? I am somewhat skeptical, but I don't know for sure.

Is vmtouch really going to solve that problem? For starters, smartctl maps some libraries that'd need to be kept resident as well in whole:

```
	linux-vdso.so.1 (0x00007ffd737b6000)
	libselinux.so.1 => /lib/x86_64-linux-gnu/libselinux.so.1 (0x000077a8d16a8000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x000077a8d1400000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x000077a8d13d2000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x000077a8d1000000)
	libpcre2-8.so.0 => /lib/x86_64-linux-gnu/libpcre2-8.so.0 (0x000077a8d1338000)
	/lib64/ld-linux-x86-64.so.2 (0x000077a8d17fc000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x000077a8d124f000)
```

Not to mention, the shell (bash) that this is probably run from, and swap, which can get involved for mutable memories, and that messes with a block device as well.

Could vmtouch make this somewhat better? Perhaps, but it seems incredibly marginal.